agent/config/config.go

```diff
@@ -86,6 +86,19 @@ const (
 
 	// DefaultMinSupportedCNIVersion denotes the minimum version of cni spec required
 	DefaultMinSupportedCNIVersion = "0.3.0"
+
+	// pauseContainerTarball is the path to the pause container tarball
+	pauseContainerTarballPath = "/images/amazon-ecs-pause.tar"
```

Comment:
nit: Can Linux-specific paths be moved to a conditionally-compiled file for Linux?